desktop/src/main/java/bisq/desktop/main/overlays/windows/DaoLaunchWindow.java

```diff
@@ -127,8 +127,7 @@ protected void onShow() {
     protected void addButtons() {
         super.addButtons();
 
-        closeButton.setPrefWidth(162);
-        actionButton.setPrefWidth(162);
+        closeButton.prefWidthProperty().bind(actionButton.widthProperty());
```

Comment:
Are you sure the setters are never called in any other popup? It would cause an error if binding is set and we call setWidth somewhere else....

Ah its not in the super class, so should be isolated then to that window.

utACK

Reply:
Yes, it was just to have the same button width based on Pedro's design for this popup.

Reply:
But still to support bigger button sizes for different languages